### PR TITLE
 Add a script to update the footer automatically

### LINKED ISSUE #838: Update footer for the year 2021
**Is your feature request related to a problem?**
- Current footer copyright contains for the year 2020.

**Describe the solution you'd like**
- It should be updated as '2021'

### PR DESCRIPTION
**Pull request by gajindubandara**
## Purpose
The purpose of this PR is to fix #838
## Goals
To update the footer automatically

## Approach
Added a script to update the year automatically

### Screenshots
Screenshot from 2021-02-01 13-05-23
### Preview Link

https://pr-847-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: assets/content/static/footer.html [hunk 1]
```diff
@@ -45,7 +45,8 @@ <h4 class="mb-0 font-weight-light">Get in touch with using any of these platform
                 <div class="col-md-6">
                     <div class="copyright">
                         &copy;
-                        2020
+
```
Comment (Gimhan-minion) [inline assets/content/static/footer.html]: You can remove the blank line in here

File: assets/content/static/footer.html [hunk 2]
```diff
@@ -73,3 +74,6 @@ <h4 class="mb-0 font-weight-light">Get in touch with using any of these platform
         </div>
     </footer>
 </div>
+<script>
+    document.getElementById('year').innerHTML = (new Date().getFullYear());
+</script>
```
Comment (Gimhan-minion) [inline assets/content/static/footer.html]: Add a new line at the end of the file

**Review by Gimhan-minion — CHANGES_REQUESTED** [review]

File: assets/content/static/footer.html (hunk 1 above)
Comment (gajindubandara) [reply assets/content/static/footer.html]: Done!

File: assets/content/static/footer.html (hunk 2 above)
Comment (gajindubandara) [reply assets/content/static/footer.html]: Done!

**Review by Gimhan-minion — APPROVED** [review]
Good Job @gajindubandara :100:

**Review by anjula-sack — APPROVED** [review]
Awesome! @gajindubandara

**Pull request merged**